app/src/main/res/drawable/ic_storage.xml

```diff
@@ -0,0 +1,13 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:autoMirrored="true">
+  <path
+      android:fillColor="?primaryText"
+      android:pathData="M2,20h20v-4L2,16v4zM4,17h2v2L4,19v-2zM2,4v4h20L22,4L2,4zM6,7L4,7L4,5h2v2zM2,14h20v-4L2,10v4zM4,11h2v2L4,13v-2z"/>
+</vector>
```

Comment:
Oh, we need to duplicate this? :(

Reply:
No really, I put it as place holder for the animated version, similar to [other permissions icon](https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/drawable/ic_location.xml), I was trying to create a strike through version, but  I'm not that good with designing tools 🦄. I followed @NotWoods 's [blog](https://medium.com/firefox-mobile-engineering/animating-a-strike-through-on-android-with-animated-vector-drawable-and-animatedstatelistdrawable-a77e66f9790f) but I was spending a lot time with [Shape Shifter](https://shapeshifter.design/), without much progress :(